pkg/cmd/task/describe_test.go

```diff
@@ -238,6 +238,82 @@ Taskruns
 	test.AssertOutput(t, expected, out)
 }
 
+func TestTaskDescribe_NoStepName(t *testing.T) {
+	clock := clockwork.NewFakeClock()
+	cs, _ := test.SeedTestData(t, pipelinetest.Data{
+		Tasks: []*v1alpha1.Task{
+			tb.Task("task-1", "ns",
+				tb.TaskSpec(
+					tb.Step("", "mandriva"),
+				),
+				tb.TaskSpec(
+					tb.Step("alatouki", "stampede"),
+				),
+				tb.TaskSpec(
+					tb.Step("", "mandrake"),
+				),
+				tb.TaskSpec(
+					tb.Step("lamaracarena", "yggdrasil"),
+				),
+			),
+		},
+		TaskRuns: []*v1alpha1.TaskRun{
+			tb.TaskRun("tr-1", "ns",
```

Comment:
Why do we create a `TaskRun` here, it's not referering our `Task`, so it's just noise in the test :stuck_out_tongue_closed_eyes:

Reply:
whateverrr.. 🤷‍♂